PKPD/gui/mainWindow.py

```diff
@@ -142,8 +142,10 @@ def next_tab(self):
         """Switches to the simulation tab, when triggered by clicking the 'next' QPushButton on the home tab.
         """
         #TODO: refactor this construction when structure of webApp is clear.
```

Comment:
Unless your IDE does something special with "#TODO" as opposed to "# TODO", it seems best to have spaces (complies with PEP8).

Reply:
agreed

Reply:
agreed.